<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Propietario</class>
 <widget class="QMainWindow" name="Propietario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administración vívero</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 203, 203);</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelPropietario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;PROPIETARIO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxFinca">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>661</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(223, 223, 223);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutFinca">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCultivo">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Cultivo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMunicipio">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Municipio:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMunicipio"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelRegCastral">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Registro Castral:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditRegCastral"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxCultivo">
        <item>
         <property name="text">
          <string>Seleccione el cultivo...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tomates</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Naranjas</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cebollas</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonAgregarFinca">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>180</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(193, 193, 193);</string>
     </property>
     <property name="text">
      <string>Agregar Finca</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxPropietario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>661</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
background-color: rgb(223, 223, 223);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonAgregarProp">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>170</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(193, 193, 193);</string>
     </property>
     <property name="text">
      <string>Agregar Propietario</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutPropietario">
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditTel"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditIdentidad"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelApellido">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Apellido:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelIdentidad">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Documento de Identidad:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Tel/Cel:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelNombre">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditNombre"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelCorreo">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Correo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditApellido"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditCorreo"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="labelFinca">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>46</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;FINCA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
